app/helpers/application_helper.rb

```diff
@@ -77,7 +77,7 @@ def script_asset_path(script)
         path = path.gsub(/\.([^.]+)$/, '.gz.\1')
       end
 
-    elsif GlobalSetting.cdn_url&.start_with?("https") && is_brotli_req?
+    elsif GlobalSetting.cdn_url&.start_with?("https") && is_brotli_req? && Rails.env != "development"
```

Comment:
Why is this change needed?

Reply:
In the development environment, we're not using/generating brotli assets. So we must skip this step in the dev environment.